providers/default.rb

```diff
@@ -234,6 +234,7 @@
 
   directory "#{new_resource.base_dir}/#{new_resource.instance_name}" do
```

Comment:
No Chef will only check that they are Strings. Added that check here, https://github.com/cerner/cerner_tomcat/pull/10/commits/ed2794950ab15d8cd12ca5818c567ccca4960e08